frontend/pages/admin/campaigns/index.tsx

```diff
@@ -1,24 +1,72 @@
-import useAdminLoginCheck from '../../../hooks/useAdminLogInCheck';
+import AddIcon from '@mui/icons-material/Add';
+import { Paper, Typography } from '@mui/material';
 import { Box, Stack } from '@mui/system';
+import moment from 'moment';
 import Head from 'next/head';
-import { Typography } from '@mui/material';
-import React from 'react';
-import { headerSx, rootSx } from '../../../styles/admin/campaigns/indexStyles';
-import Button from '../../../components/generic/Button';
 import { useRouter } from 'next/router';
-import AddIcon from '@mui/icons-material/Add';
+import useSWR from 'swr';
+import Button from '../../../components/generic/Button';
+import SimpleTable from '../../../components/generic/SimpleTable';
+import IconButtonWithTooltip from '../../../components/IconButtonWithTooltip';
+import Tabbed from '../../../components/Tabs';
+import api from '../../../frontendApis';
+import CampaignsAPI from '../../../frontendApis/campaigns';
+import useAdminLoginCheck from '../../../hooks/useAdminLogInCheck';
+import { headerSx, rootSx } from '../../../styles/admin/campaigns/indexStyles';
+import { CampaignAdminListData } from '../../../types/campaigns';
+import { Nullable } from '../../../types/utils';
+import { DATE_FORMAT } from '../../../utils/constants';
+import EditIcon from '@mui/icons-material/Edit';
 
 const AdminCampaigns = () => {
   useAdminLoginCheck();
+  const { data: campaigns } = useSWR<Nullable<CampaignAdminListData[]>>(CampaignsAPI.CAMPAIGNS_URL, () =>
+    api.campaigns.adminList().then((r) => r.payload),
+  );
   const router = useRouter();
 
+  const upcomingCampaigns = campaigns?.filter((c) => c.start.isAfter(moment.now()));
+  const activeCampaigns = campaigns?.filter((c) => c.start.isBefore(moment.now()) && c.end.isAfter(moment.now()));
+  const completedCampaigns = campaigns?.filter((c) => c.end.isBefore(moment.now()));
+
+  const makeCampaignsTable = (campaigns: Nullable<CampaignAdminListData[]> | undefined) => {
+    const goToManageCampaignAction = {
+      component: <IconButtonWithTooltip icon={<EditIcon />} tooltip="Manage Campaign" />,
```

Comment:
Up to you but maybe we can use this https://mui.com/material-ui/material-icons/?query=manage&selected=ManageAccounts as the button doesn't really go to an edit page